bin/adhoc-scripts/parseUnifiedCampaigns.py

```diff
@@ -276,14 +311,24 @@ def main():
             key = proxy
         hdict = {'cert': cert, 'key': key, 'pycurl': True}
         mgr = ReqMgrAux(opts.url, hdict, logger=logger)
-    if opts.dburi:
+    if opts.cin:
+        # then simply use this json file where campaign configurations are
+        # already in the WMCore format
+        with open(opts.cin) as jo:
+            data = json.load(jo)
+        print("About to insert %d campaign documents into CouchDB..." % len(data))
+        for rec in data:
+            upload(mgr, rec)
+        insertTestCampaigns(mgr)
+        sys.exit(0)
```

Comment:
Do you really need sys.exit here? this is inside of if block already and no other action should be executed.